packages/legend-application-studio/src/components/editor/editor-group/function-activator/testable/FunctionTestableEditor.tsx

```diff
@@ -988,8 +997,10 @@ const FunctionTestSuiteEditor = observer(
 );
 
 const CreateFucntionTestSuiteModal = observer(
-  (props: { functiontestableEditorState: FunctionTestableState }) => {
-    const { functiontestableEditorState } = props;
+  (props: { functionTestableEditorState: FunctionTestableState }) => {
+    const { functionTestableEditorState: functiontestableEditorState } = props;
```

Comment:
could we  make corresponding changes to delete this `const { functionTestableEditorState: functiontestableEditorState } = props;`

Reply:
Good catch! I've fixed this in the new MR: https://github.com/finos/legend-studio/pull/3019